Modules/_tkinter.c

```diff
@@ -885,8 +887,8 @@ asBignumObj(PyObject *value)
     PyObject *hexstr;
     const char *hexchars;
     mp_int bigValue;
-
-    neg = Py_SIZE(value) < 0;
+    assert(PyLong_Check(value));
+    neg = _PyLong_IsNegative((PyLongObject *)value);
```

Comment:
Please put the blank line back.
```suggestion

    assert(PyLong_Check(value));
    neg = _PyLong_IsNegative((PyLongObject *)value);
```